tex/alg-NT/ramification.tex

```diff
@@ -174,10 +175,34 @@ \section{The magic of Galois extensions}
 \begin{moral}
 	All of the $\{\kp_i\}$ are Galois conjugates of each other.
 \end{moral}
+
+Before proving this, let us consider the easier problem
+of factorization into elements.
+\begin{quote}
+	Suppose $\OO_K$ is an UFD, and $p$ factors as $u p_1 p_2 \cdots p_n$ in
+	$\OO_K$, where $p_i$ are irreducibles and $u$ is an unit.
+	Show that the $p_i$ are all conjugates of each other, up to
+	multiplication by an unit.
+\end{quote}
+\begin{ques}
+	Try to prove it before reading it below.
+	(Hint: Galois theory. Alternatively, take the norm of $p_1$.)
+\end{ques}
 \begin{proof}
-	Fairly slick.
-	Suppose for contradiction that no $\sigma \in G$ sends $\kp_1$ to $\kp_2$, say.
-	By the Chinese remainder theorem, we can find an $x \in \OO_K$ such that
+	Let $q=\NK(p_1)$ be the product of all conjugates
+	of $p_1$, then $q \in \QQ$. Thus $p \mid q$, so each $p_i$ is a factor of
+	$q$, and we're done by unique factorization.
+\end{proof}
+
+Unfortunately, the product of all conjugates of an ideal $\kp_1$ is not
+necessarily of the form $p \cdot \OO_K$ (for example, $K=\QQ[i]$ and $(1+i)$
+has no other conjugates). So in the proof, we pick $x$ which is an
+``representative'' of $\kp_1$.
+
+\begin{proof}
+	[Proof of \Cref{thm:galois_group_transitive}]
+	Because $\kp_i$ are distinct primes, by the Chinese remainder theorem,
+	we can find an $x \in \OO_K$ such that
```

Comment:
I agree that the original proof is slick, but I realize that after reading (skimming through?) it once I forgot it instantly. So I try to give some intuition on where it comes from.